sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/PatchAsyncTest.java

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.models.CosmosItemResponse;
+import com.azure.cosmos.models.PartitionKey;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+import reactor.core.publisher.Flux;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PatchAsyncTest extends BatchTestBase {
+    private CosmosAsyncClient client;
+    private CosmosAsyncContainer container;
+
+    @Factory(dataProvider = "clientBuilders")
+    public PatchAsyncTest(CosmosClientBuilder clientBuilder) {
+        super(clientBuilder);
+    }
+
+    @BeforeClass(groups = { "emulator" }, timeOut = SETUP_TIMEOUT)
+    public void before_PatchAsyncTest() {
```

Comment:
we don't add "Test" as prefix nor suffix to the test methods. Please remove the suffix for all the test methods here and elsewhere.